lib/phobos/actions/process_message.rb

```diff
@@ -64,6 +62,21 @@ def process_message(payload)
           end
         end
       end
+
+      def wait_for_next_retry(interval)
+        MAX_SLEEP_INTERVAL = 3
```

Comment:
Constant definition inside of methods isn't allowed in Ruby.  Could just move this into the class definition instead.